requirements/base.txt

```diff
@@ -5,26 +5,26 @@
 #    make upgrade
 #
 certifi==2020.6.20        # via elasticsearch, requests
-cffi==1.14.0              # via cryptography
+cffi==1.14.2              # via cryptography
 chardet==3.0.4            # via requests
 coreapi==2.3.3            # via django-rest-swagger, openapi-codec
 coreschema==0.0.4         # via coreapi
-cryptography==2.9.2       # via pyjwt
-django-cors-headers==3.4.0  # via -r requirements/base.in
+cryptography==3.1         # via pyjwt
```

Comment:
https://cryptography.io/en/latest/changelog/#master says BACKWARDS INCOMPATIBLE, but I think it's ok if we don't use `idna based U-label parsing in various X.509 classes.`:

![image](https://user-images.githubusercontent.com/278814/93374000-2286eb80-f824-11ea-8465-b517207ffdff.png)